src/Button/Button.js

```diff
@@ -33,76 +44,119 @@ export const Button = ({ className, children, onClick, ref, variant, size, type,
     }
   }
 
-  // Props.size
+  // size
   if (size && buttonSizes.includes(size)) {
-    switch (size) {
-      case 'big':
-        levers.size = 'su-px-34 su-py-15 su-text-20 md:su-text-24';
-        break;
+    levers.size = getButtonSize(size);
+  }
```

Comment:
Pulling button sizes into common levers/functions as well; I might do that with the variants also.